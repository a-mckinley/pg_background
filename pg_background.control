comment = 'Run SQL queries in the background'
default_version = '1.1'
module_pathname = '$libdir/pg_background'
relocatable = true
