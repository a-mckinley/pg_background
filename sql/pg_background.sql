CREATE EXTENSION pg_background;

CREATE TABLE t(id integer);

SELECT * FROM pg_background_result(pg_background_launch('INSERT INTO t SELECT 1')) AS (result TEXT);

SELECT * FROM t;
