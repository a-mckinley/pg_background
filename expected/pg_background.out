CREATE EXTENSION pg_background;
CREATE TABLE t(id integer);
SELECT * FROM pg_background_result(pg_background_launch('INSERT INTO t SELECT 1')) AS (result TEXT);
   result   
------------
 INSERT 0 1
(1 row)

SELECT * FROM t;
 id 
----
  1
(1 row)

